conformance/vfs_assignement_test.go

```diff
@@ -0,0 +1,189 @@
+			Expect(err).ToNot(HaveOccurred(), "No SR-IOV supported devices detected")
+			Expect(intf.TotalVfs).To(BeNumerically(">", 2), "Cluster has less than 2 VFs available.")
+
+			nodePolicyConfig = &sriovv1.SriovNetworkNodePolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "testpolicy",
+					Namespace:    operatorNamespace,
+				},
+
+				Spec: sriovv1.SriovNetworkNodePolicySpec{
+					NodeSelector: map[string]string{
+						"kubernetes.io/hostname": testNode,
+					},
+					NumVfs:       2,
+					ResourceName: "testresource",
+					Priority:     99,
+					NicSelector: sriovv1.SriovNetworkNicSelector{
+						PfNames: []string{intf.Name},
+					},
+					DeviceType: "netdevice",
+				},
+			}
+			err = clients.Create(context.Background(), nodePolicyConfig)
+			Expect(err).ToNot(HaveOccurred(), "Error to create SriovNetworkNodePolicy")
+
+			Eventually(func() sriovv1.Interfaces {
+				nodeState, err := clients.SriovNetworkNodeStates(operatorNamespace).Get(testNode, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return nodeState.Spec.Interfaces
+			}, 1*time.Minute, 1*time.Second).Should(ContainElement(MatchFields(
+				IgnoreExtras,
+				Fields{
+					"Name":     Equal(intf.Name),
+					"NumVfs":   Equal(2),
+					"VfGroups": ContainElement(sriovv1.VfGroup{ResourceName: "testresource", DeviceType: "netdevice", VfRange: "0-1"}),
+				})), "Error SriovNetworkNodeState doesn't conatin reqired elements")
+			Eventually(func() int64 {
+				testedNode, err := clients.Nodes().Get(testNode, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				resNum, _ := testedNode.Status.Capacity["openshift.io/testresource"]
+				capacity, _ := resNum.AsInt64()
+				return capacity
+			}, 3*time.Minute, time.Second).Should(Equal(int64(2)), "Error discovered less than 2 VFs")
+
+			networkConfig = &sriovv1.SriovNetwork{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "testnetwork",
+					Namespace:    operatorNamespace,
+				},
+				Spec: sriovv1.SriovNetworkSpec{
+					NetworkNamespace: namespaces.Test,
+					Vlan:             0,
+					IPAM: `{ "type": "host-local","subnet": "10.10.10.0/24","rangeStart": 
+					"10.10.10.171","rangeEnd": "10.10.10.181","routes": [{ "dst": "0.0.0.0/0" }],"gateway": "10.10.10.1"}`,
+					ResourceName: nodePolicyConfig.Spec.ResourceName,
+				},
+			}
+			err = clients.Create(context.TODO(), networkConfig)
+			Expect(err).ToNot(HaveOccurred(), "Error to create SriovNetwork")
+			netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
+			Eventually(func() error {
+				return clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: networkConfig.Name, Namespace: namespaces.Test}, netAttDef)
+			}, 30*time.Second, 1*time.Second).ShouldNot(HaveOccurred(), "Error to detect NetworkAttachmentDefinition")
+		})
+
+		Context("Virtual Functions", func() {
+			// 21396
+			It("The VFs should be released once the pod deleted and can be used by the new created pods", func() {
```

Comment:
It (should release the vfs once ..) produce a more readable output

Reply:
I added Should release the VFs once the pod deleted and same VFs can be used by the new created pods. I hope it's more readable